### DISCUSSION TITLE
Android TV support

### DISCUSSION THREAD

**Discussion by kid1412621** [trakt/showly#667] [Ideas / Feature Requests]
or any larger screen devices running Android

**Comment by larsjarred9** [comment]
Hi @kid1412621 You can already link slowly with Trakt.tv and they offer some support on android tv's

**Reply by kid1412621** [reply]
> Hi @kid1412621 You can already link slowly with Trakt.tv and they offer some support on android tv's

Thanks, but I prefer Showly as a standalone Android TV app to use.

**Comment by KingLucius** [comment]
This one is important, +1 for it
